sql/sql.y

```diff
@@ -175,18 +176,17 @@ select
 train_clause
 : TRAIN IDENT WITH attrs column_clause label_clause INTO IDENT {
 	$$.estimator = $2
-	$$.attrs = $4
+	$$.trainAttrs = $4
 	$$.columns = $5
 	$$.label = $6
 	$$.save = $8
   }
 ;
 
 predict_clause
-: PREDICT IDENT USING IDENT {
-	$$.into = $2
-	$$.model = $4
-}
+: PREDICT IDENT { $$.into = $2 }
+| PREDICT WITH attrs { $$.predAttrs = $3 }
+| predict_clause USING IDENT { $$.model = $3 } 
```

Comment:
Does this line mean we can write multiple `USING`?

Reply:
Done, and found some same syntax bug, such as `SELECT xx FROM a FROM b`, `SELECT xxx from table TRAIN model COLUMN a,b COLUMN c,d`.